frontend/src/components/Map/EditMapButton.tsx

```
@@ -0,0 +1,21 @@
+  mapId: number;
+  text: string;
+};
+
+const EditMapButton = ({ mapId, text }: EditMapButtonProps) => {
+  const navigate = useNavigate();
+
+  const onClick = () => {
+    navigate(`/edit/map/${mapId}`);
+  };
+  return (
+    <button onClick={onClick} className="text-xs text-c_placeholder_gray">
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)